src/bin/lib/parson/tests/test_2.txt

```diff
@@ -1,4 +1,4 @@
-﻿{
+{
```

Comment:
Not our file, can you remove that change from the PR?

Reply:
This was my mistake. I verified that this PR now has the minimal set of changes required to pass the current editorconfig rules.

`src/bin/lib/.editorconfig` file allows us to ignore all the files under `/src/bin/lib` now.

I removed this change from the PR.